scss/base/_global.scss

```
@@ -76,12 +76,35 @@ h6 {
 .wrapper {
   max-width: 112rem;
   margin: 0 auto;
+  @include respond-to('tablet') {
+    padding-inline: 3rem;
+  }
   @include respond-to('mobile-and-down') {
     padding-inline: 1rem;
   }
 }
-button {
+button,
+.btn {
   border: none;
+  outline: none;
+  background-color: $white;
+  &--primary {
```

```suggestion
  background-color: $white;
  
  &--primary {
```
Mỗi block ngăn cách nhau 1 dòng nhé. Kèm theo đưa `@include` lên trên đầu nhé